NGRValidator/NGRValidator/Categories/NSData+NGRValidator.h

```diff
@@ -0,0 +1,23 @@
+//
+//  NSData+NGRValidator.h
+//  NGRValidator
+//
+//
+//
+
+#import "NGRMimeType.h"
+
+@interface NSData (NGRValidator)
+
+NS_ASSUME_NONNULL_BEGIN
+
+- (BOOL)ngr_hasMimeType:(NGRMimeType *)type;
+
+- (BOOL)ngr_isImage;
```

Comment:
Well, yes. But I think you can start right away by adding that as an example for the future.